upgrade_tests/helpers/existence.py

```diff
@@ -189,14 +189,13 @@ def set_datastore(datastore, endpoint):
         raise IncorrectEndpointException(
             'Endpoints has to be one of {}'.format(allowed_ends))
     if endpoint == 'cli':
-        org = os.environ.get('ORGANIZATION', 'Default_Organization')
         nonorged_comps_data = [
             csv_reader(
                 component, 'list') for component in cli_const.components[
                 'org_not_required']]
         orged_comps_data = [
             csv_reader(
-                component, 'list --organization {}'.format(org)
+                component, 'list --organization-id 1'
```

Comment:
@san7ket , Now hardcoding that to org-id 1, once I implement this -> https://github.com/SatelliteQE/satellite6-upgrade/issues/4.

I can accept and calculate the org-id according to the org-name provided.

Reply:
ACK